lib/domain/models/memo.dart

```diff
@@ -10,6 +10,7 @@ import 'package:meta/meta.dart';
 /// infinite amount of times, its purpose is to store the most recent version of its question/answer with useful
 /// execution's metadata, like [lastExecution] and by extending [MemoExecutionsMetadata].
 @immutable
+// ignore: avoid_implementing_value_types
```

Comment:
Is this rule necessary?

Reply:
I think they are really useful in scenarios one can't run away from mutable-forced classes, like this one